design/src/genre-analysis.md

```diff
@@ -0,0 +1,121 @@
+# Genre Analysis: Factory Builders
+
+## Defining Games
+
+- Factorio
+  - colonize an alien planet and launch a rocket
+- Satisfactory
+  - open world first person factory building
+- Shapez.io
+  - abstract factory puzzles
+- Dyson Sphere Program
+  - gather resources from an entire solar system
+
+## Defining characteristics
+
+- automated resource extraction
```

Comment:
Great questions! I'll try and chew on that.

> I wonder because of the comparison to idlers later, where the automation in general isn't player designed, right?

Precisely: games like Forager and Stardew Valley and Cookie Clicker don't have intricate automation pipelines that offer an opportunity for creativity.